e2e/testdata/expected-image-ls.golden

```diff
@@ -0,0 +1,3 @@
+REFERENCE           NAME   VERSION
```

Comment:
Can you split the `REFERENCE` column into `REPOSITORY` and `TAG` columns to better fit with `docker image ls` output?